src/js/coblocks-events.js

```diff
@@ -1,18 +1,45 @@
-/*global jQuery */
-( function( $ ) {
+import TinySwiper from 'tiny-swiper';
+import TinySwiperPluginNavigation from 'tiny-swiper/lib/modules/navigation.min.js';
+
+( function() {
 	'use strict';
 
-	$( document ).ready( function() {
-		const calendars = $( '.wp-block-coblocks-events[data-per-page]' );
-
-		if ( calendars ) {
-			calendars.each( function() {
-				$( this ).slick( {
-					infinite: false,
-					rows: this.dataset.perPage,
-					waitForAnimate: false,
-				} );
-			} );
-		}
-	} );
-}( jQuery ) );
+	document.addEventListener( 'DOMContentLoaded', function() {
+		setTimeout( () => {
+			const frontEndContainers = document.querySelectorAll( '.page .wp-block-coblocks-front-events-swiper-container' );
+
+			// there may be multiple event blocks
+			for ( let j = 0; j < frontEndContainers.length; j++ ) {
+				const frontEndContainer = frontEndContainers[ j ];
+
+				if ( frontEndContainer ) {
```

Comment:
@snovosel I would reverse this condition and return early. It removes 1 level of indentation and makes it easier to read.